### DISCUSSION TITLE
Hotkeys Customization

### DISCUSSION THREAD

**Discussion by v-i-m** [sumatrapdfreader/sumatrapdf#4039] [Ideas]
Hi all,

I've been using Sumatra for a while now, I love it. I just have one issue with it. There is no simple and reliable way to customize hotkeys. I don't think it's even possible to remove default hotkeys, pls correct me if I'm wrong.

For example, I tried to remove 'Space' from CmdScrollDownPage. I've added a Key = 1, for example, as a test, in the settings txt file, and while you can add new hotkeys there, it seems you cannot override the defaults. So 'Space', in this case will still CmdScrollDownPage, along with the newly added '1'. Am I doing it wrong?

In any case, it's not a good nor simple way to customize hotkeys anyway, and I think this is an important feature of any software. It would be ideal to allow users to customize hotkeys easily, reliably, and transparently, with a simple, clear interface. So in this case I'd want to see that 'Ctrl + Down', 'PageDown', 'Return', and 'Space' are all default hotkeys for the command 'Scroll Down Page', and I'd like to be able to clear all that and set my own hotkey/s.

Cheers,
VM

**Comment by GitHubRulesOK** [comment]
set space will be a problem as it is no distinct character so in the manual it is "Special Key"

```
Shortcuts [
	[
		Cmd = CmdNone
		Key = Space
	]
]
```

**Reply by v-i-m** [reply]
Works great, thanks!
